public/app/features/logs/components/LogRow.tsx

```diff
@@ -92,6 +92,11 @@ class UnThemedLogRow extends PureComponent<Props, State> {
       return;
     }
 
+    // If the user selected some text, we don't want to toggle the details.
+    if (window.getSelection()?.toString()) {
```

Comment:
This is great at getting the selected text in the browser, but it has the consequence in this situation that you cannot expand/collapse when any text is selected.

Also when you have text selected, but you aren't clicking on that selected text, but you aren't clicking on/selecting other text (i.e. you are clicking on the whitespace outside the log text), 
<img width="1623" alt="image" src="https://github.com/grafana/grafana/assets/109082771/ce091446-7fef-4e52-a1d7-340f68537aca">
The UI falls into this state where you have to unselect your text or move your cursor to trigger the collapse. This feels very frustrating as a user because you have to shift your focus and only click on the text.

Reply:
Seems to be releated with the `wrap lines option`, otherwise the selection is removed when you click somewhere else. I think I recall that you mentioned that we recently changed a css property related with it?

Reply:
Scratch that. It's related with wrap lines, but what changes is the size of the button that displays the log line. When wrap is _disabled_, it is as long as the text. When wrap is _enabled_, it becomes larger so you always click on it.